rosidl_runtime_cpp/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosidl_runtime_cpp</name>
+  <version>0.8.2</version>
+  <description>Generate the ROS interfaces in C++.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <buildtool_export_depend>ament_cmake</buildtool_export_depend>
+
+  <exec_depend>rosidl_parser</exec_depend>
```

Comment:
Wasn't the goal for this package to *not* depend on Python stuff? Also the package shouldn't need this.